### DISCUSSION TITLE
SDXL second pass refiner. Best settings?

### DISCUSSION THREAD

**Discussion by ASeale** [vladmandic/sdnext#1710] [Q&A]
I've tried changing every setting in Second Pass and every image comes out looking like garbage.

**Comment by Centurion-Rome** [comment]
Have the same + **performance dropped significantly since last update(s)** !

Lowering Second pass Denoising strength to about 0.23-0.25 and refiner steps count to be max 30-30% of step from base
did some improvements but still not the best output as compared to some previous commits :(

**Comment by vladmandic** [comment]
that's a long discussion in discord and we're still tweaking. don't want to duplicate here, especially since all sd-xl related items are explicitly stated to go into discord while its experimental.

**Reply by ASeale** [reply]
My apologies, I missed that statement.
